src/lib/Permit.sol

```diff
@@ -0,0 +1,13 @@
+// SPDX-License-Identifier: GPL-3.0-or-later
+pragma solidity ^0.8.13;
+
+/// @dev EIP-2612 Permit
+struct Permit {
+    uint256 value;
+    uint256 deadline;
+    uint8 v;
+    bytes32 r;
+    bytes32 s;
```

Comment:
I don't understand what 'v', 'r' and 's' mean.  Can these variables be renamed?

Reply:
They are the components of the ECDSA signature. I've always seen then referenced this way.

Reply:
@xPi2 Then we can add something like "The components of the ECDSA signature" to the documentation 😄